src/overlays/actors/ovl_En_Paper/z_en_paper.c

```diff
@@ -151,8 +196,8 @@ void EnPaper_Update(Actor* thisx, PlayState* play) {
     EnPaper* this = THIS;
 
     this->actionFunc(this, play);
+    EnPaper_UpdateWind(this);
 
-    func_80C1F87C(this);
     if (this->timer == 0) {
         Actor_MarkForDeath(&this->actor);
     } else {
```

Comment:
```suggestion
        Actor_MarkForDeath(&this->actor);
        return;
    }
```
Since it is an `Actor_MarkForDeath`